src/pyvcam_driver.py

```diff
@@ -0,0 +1,537 @@
+        """
+        Returns a dictionary containing the port, speed and gain table, 
+        which gives information such as bit depth and pixel time for each readout port, speed index and gain.
+
+        :return:
+        """
+        return self.cam.port_speed_gain_table
+
+    def readout_port(self) -> int:
+        """
+        Some cameras may have many readout ports, which are output nodes from which a pixel stream can be read from. 
+        For more information about readout ports, refer to the Port and Speed Choices section inside the PVCAM User Manual
+
+        :return:
+        """
+        return self.cam.readout_port
+
+    def set_readout_port(self, value: int) -> None:
+        """
+        Changes readout port. By default only one available value (0).
+
+        :param value: Desired readout port.
+        :return: None.
+        """
+        self.cam.readout_port = value
+
+    def serial_no(self) -> str:
+        """
+        Returns the camera's serial number as a string.
+
+        :return:
+        """
+        return self.cam.serial_no
+    
+    def get_sequence(self, num_frames: int, exp_time: int=None, 
+                     timeout_ms: int=WAIT_FOREVER, interval: int=None) -> list[list[list[int]]]:
+        """
+        Calls the pvc.get_frame function with the current camera settings in
+        rapid-succession for the specified number of frames
+
+        :param num_frames: Number of frames to capture in the sequence.
+        :param exp_time: The exposure time.
+        :param interval: The time in milliseconds to wait between captures.
+
+        :return: A 3D np.array containing the pixel data from the captured frames.
+        """
+        return self.cam.get_sequence(num_frames, exp_time, timeout_ms, interval)
+    
+    def start_live(self, exp_time: int=None, buffer_frame_count: int=16,
+                   stream_to_disk_path: str=None) -> None:
+        """
+        Calls the pvc.start_live function to setup a circular buffer acquisition.
+
+        :param exp_time: The exposure time.
+        :param buffer_frame_count: The number of frames in the circular frame buffer.
+        :param stream_to_disk_path: The file path for data written directly to disk by PVCAM. 
+                                    The default is None which disables this feature.
+
+        :return: None.
+        """
+        self.cam.start_live(exp_time, buffer_frame_count, stream_to_disk_path)
+    
+    def start_seq(self, exp_time: int=None, num_frames: int=1) -> None:
+        """
+        Calls the pvc.start_seq function to setup a non-circular buffer acquisition.
+        This must be called before poll_frame.
+
+        :param exp_time: The exposure time.
+
+        :return: None.
+        """
+        self.cam.start_seq(exp_time, num_frames)
+    
+    def poll_frame(self, timeout_ms: int=WAIT_FOREVER, oldestFrame: bool=True, 
+                   copyData: bool=True) -> tuple[dict, float, int]:
```

Comment:
```suggestion
    def poll_frame(self, timeout_ms: int=WAIT_FOREVER, oldest_frame: bool=True, copy_data: bool=True) -> tuple[dict, float, int]:
```
Formatting. Also, follow Python variable name convention.